cardano_node_tests/tests/test_native_tokens.py

```diff
@@ -2127,6 +2204,30 @@ def test_transfer_invalid_token_amount(
             exc_val = str(excinfo.value)
             assert "ValueNotConservedUTxO" in exc_val, exc_val
 
+        elif build_method == clusterlib_utils.BuildMethods.BUILD_EST:
+            with pytest.raises(clusterlib.CLIError) as excinfo:
+                try:
+                    logging.disable(logging.ERROR)
+                    cluster.g_transaction.build_estimate_tx(
+                        src_address=src_address,
+                        tx_name=temp_template,
+                        txouts=txouts,
+                        tx_files=tx_files,
+                    )
+                finally:
+                    logging.disable(logging.NOTSET)
+
+            exc_val = str(excinfo.value)
+            # TODO: refine once CLI issue #1199 is fixed
+            # At this point we don't know the exact error string from build-estimate.
+            # Let this fail if it's anything unexpected so we can revisit later.
+            assert (
+                "balance" in exc_val or "ValueNotConservedUTxO" in exc_val or "UTxO" in exc_val
```

Comment:
We don't know yet how the actual error for `build-estimate` looks like, because it is masked by cli issue 1199.
At the same time we want to make sure we are matching **just** the expected error message.
Checking if the error contains "balance" will likely match all sorts of error and lead to false positives. Let's check `TODO: unknown atm` and replace it with precise error string as soon as it starts failing.